src/TicketingSystem.EventsApi/Controllers/EventsController.cs

```diff
@@ -0,0 +1,115 @@
+    /// </summary>
+    [ApiController]
+    [Route("[controller]")]
+    public class EventsController : ControllerBase
+    {
+        private readonly IEventService _eventService;
+        private readonly IEventSeatService _eventSeatService;
+        private readonly IEventSectionService _eventSectionService;
+        private readonly IMapper _mapper;
+
+        public EventsController(
+            IEventService eventService,
+            IEventSeatService eventSeatService,
+            IEventSectionService eventSectionService,
+            IMapper mapper)
+        {
+            _eventService = eventService;
+            _eventSeatService = eventSeatService;
+            _eventSectionService = eventSectionService;
+            _mapper = mapper;
+        }
+
+        /// <summary>
+        /// Returns a list of Venues with their info (no additional entities attached)
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<IActionResult> GetEvents()
+        {
+            var events = await _eventService.GetAllAsync();
+
+            return Ok(events);
+        }
+
+        /// <summary>
+        /// Returns a list of Event Seats (with Event Section id, Event Row Id) with their statuses and prices
+        /// </summary>
+        /// <param name="eventId"></param>
+        /// <param name="sectionId"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("{eventId}/sections/{sectionId}/seats")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetFullSeats([FromRoute] string eventId, [FromRoute] string sectionId)
+        {
+            var foundEvent = await _eventService.GetByIdAsync(eventId);
+
+            if (foundEvent is null)
+            {
+                return NotFound(nameof(eventId));
+            }
+
+            var section = await _eventSectionService.GetByIdAsync(sectionId);
+
+            if (section is null)
+            {
+                return NotFound(nameof(sectionId));
+            }
+
+            return Ok(await GetSeatsInfo(section));
+        }
+
+        private async Task<List<EventSeatInfoModel>> GetSeatsInfo(EventSectionDto section)
+        {
+            var seatsInfo = section.EventRows.SelectMany(er => er.EventSeatsIds, (er, seatId) => new
+            {
+                sectionId = section.Id,
+                sectionClass = section.Class,
+                sectionNumber = section.Number,
+                rowNumber = er.Number,
+                price = er.Price,
+                seatId
+            })
+                .ToList();
+
+            var result = new List<EventSeatInfoModel>();
+
+            foreach (var data in seatsInfo)
+            {
+                var seat = await _eventSeatService.GetByIdAsync(data.seatId);
+
```

Comment:
What if we have a section with 5000 seats - 5000 calls? this endpoint demonstrates incorrect usage of no-sql DB. it should be either a bulk call by seatIds or (wich would be more correct from my perspective) include eventId and sectionId into an eventSeat document, and then get all items by eventId and sectionId.